libraries/chain/hotstuff/test/test_hotstuff_state.cpp

```diff
@@ -19,27 +19,27 @@ const std::string file_path_1("temp_hs_safety");
 
 BOOST_AUTO_TEST_CASE(write_safety_state_to_file) try {
 
-  eosio::hotstuff::hs_proposal_message hspm_1;
-  eosio::hotstuff::hs_proposal_message hspm_2;
+  eosio::chain::hs_proposal_message hspm_1;
```

Comment:
I would just add a `using namespace eosio::chain;` at the beginning of the file.